pkg/assets/handlers.go

```diff
@@ -154,6 +154,22 @@ type WebConsoleVersion struct {
 	OpenShiftVersion  string
 }
 
+var extensionPropertiesTemplate = template.Must(template.New("webConsoleExtensionProperties").Parse(`
+window.OPENSHIFT_EXTENSION_PROPERTIES = {
+  {{ $values := .ExtensionPropertyValues }}
+  {{ range $i, $key := .ExtensionPropertyKeys }}
+  {{ if $i }},{{ end }}
+  {{ $value := index $values $i }}
+  "{{ $key | js }}": "{{ $value | js }}"
```

Comment:
This doesn't look like its doing escaping, but the doc here https://github.com/openshift/origin/pull/8934/files#diff-3788994f990386b63387ece5c3b5dc33R570 doesn't mention any need for the user to escape values himself.